### PR TITLE
kill and suspend arbitrary number of processes in tick-cluster.

### PR DESCRIPTION
**Pull request by btromanova**
This pr should solve two issues:
- currently if we try to kill or suspend more processes number of 'alive', tick-cluster will get into infinite loop.
- currently it's not possible to kill or suspend more then 9 processes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,7 @@
     "lodash": "^4.6.1",
     "strformat": "^0.0.7",
     "uber-licence": "^2.0.1",
+    "underscore": "^1.5.2",
```
Comment (Motiejus) [inline package.json]: It feels strange to have both underscore and lodash in dependencies. Can we get rid of one of them?

**Comment by mennopruijssers** [comment]
Now you need always to hit enter after a command. Is it possible to only wait for the enter when inputting a digit? 
Also, `space` to draw a horizontal line doesn't work anymore (probably because of the trim)

**Comment by btromanova** [comment]
@mennopruijssers Switched rawInputMode back to true, now there is no need for enter, but still possible to insert multidigit number of processes to kill or suspend.

File: tools/tick-cluster.js [hunk 2]
```diff
@@ -309,18 +311,21 @@ function onData(char) {
...
             state = 'top';
             func = null;
+            num = 0;
         } else {
             console.error('expecting: ' + numArgRe);
             state = 'top';
+            num = 0;
         }
     } else if (state === 'readchar') {
         if (char.match(debugRe)) {
```
Comment (mennopruijssers) [inline tools/tick-cluster.js]: It might be nice to write the char here to stdout as well.

**Comment by mennopruijssers** [comment]
LGTM!

**Comment by Motiejus** [comment]
LGTM

**Pull request merged**